kimchi/src/tests/recursion.rs

```diff
@@ -21,6 +21,7 @@ fn test_recursion() {
 
     // setup
     let test_runner = TestFramework::default()
+        .num_prev_challenges(1)
```

Comment:
I guess, I assumed that different previous proofs could have different lengths: https://github.com/o1-labs/proof-systems/pull/609/files#diff-a85ec89f2d7bea31a5c8b7d94a59a4bba4343bdfff7564a3e9a3436d740d0e6bR39

Reply:
oh no I see what you're doing, you're just checking the number of proofs to verify, not the number of challenges in each proof